packages/create-next-app/index.ts

```diff
@@ -111,22 +102,26 @@ const program = new Command(packageJson.name)
   })
   .allowUnknownOption()
   .parse(process.argv)
-  .opts()
-
-const packageManager = !!program.useNpm
-  ? 'npm'
-  : !!program.usePnpm
-    ? 'pnpm'
-    : !!program.useYarn
-      ? 'yarn'
-      : !!program.useBun
-        ? 'bun'
-        : getPkgManager()
+
+const opts = program.opts()
+const { args } = program
+
+const packageManager: PackageManager = opts.use
```

Comment:
Maybe a cleaner way to write this logic is this way?

```
const packageManager: PackageManager = opts.use 
  ?? (args.includes('--use-npm') && 'npm')
  ?? (args.includes('--use-pnpm') && 'pnpm')
  ?? (args.includes('--use-yarn') && 'yarn')
  ?? (args.includes('--use-bun') && 'bun')
  ?? getPkgManager();
 ```

Reply:
Removed the `--use` flags, left out the work of `program` to `opts`.
x-ref: https://github.com/vercel/next.js/pull/68229#discussion_r1693996728